### PR TITLE
"Nagions" should be "Nagios“

### PR DESCRIPTION
**Pull request by sunlintong**
“NRPE” refers to "Nagios Remote Execution Protocol"  :P

### CODE REVIEW

File: docs/getting-started-guides/ubuntu/monitoring.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Follow the above instructions on connect topbeat and filebeat to the application
 
 ## Connecting Nagios
 
-Nagios utilizes the Nagions Remote Execution Protocol (NRPE) as an agent on each node to derive machine level details of the health and applications.
+Nagios utilizes the Nagios Remote Execution Protocol (NRPE) as an agent on each node to derive machine level details of the health and applications.
```
Comment (zhangxiaoyu-zidif) [inline docs/getting-started-guides/ubuntu/monitoring.md]: NRPE protocol=Nagios Remote Plugin Executor protocol
here should do more modifications. =)

**Comment by sunlintong** [comment]
changed as prompt :p

**Pull request merged**